### ISSUE TITLE
core support?

### ISSUE THREAD

**Issue by TerribleDev** [StyleCop/StyleCop#126]
Hi, I recently tried using the msbuild package on a dotnet core project, and I get the following error:

Are there plans to support core?
```
C:\Users\tparnell\.nuget\packages\stylecop.msbuild\4.7.55\build\StyleCop.MSBuild.Targets(101,5): error MSB4062: The "Sty
leCopTask" task could not be loaded from the assembly C:\Users\tparnell\.nuget\packages\stylecop.msbuild\4.7.55\build\..
\tools\StyleCop.dll. Could not load file or assembly 'System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyT
oken=b77a5c561934e089'. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct,
that the assembly and all its dependencies are available, and that the task contains a public class that implements Micr
osoft.Build.Framework.ITask. [C:\projects\libtst4\libtst4.csproj]
```

**Comment by thomaslevesque** [comment]
I just hit the exact same problem. Building with MSBuild works, but `dotnet build` doesn't.

**Comment by TerribleDev** [comment]
To be clear, MSBuild on windows works. MSBuild on linux does not work, I have not tried xbuild. I suppose I could.

**Comment by thomaslevesque** [comment]
@TerribleDev I don't think xbuild will work better... Windows Forms isn't available on Linux.

**Comment by TerribleDev** [comment]
@thomaslevesque its borderline out of scope for this convo, but System.Windows.Forms does exists in mono. its not 100% complete, but its very close

 http://www.mono-project.com/docs/gui/winforms/
https://github.com/mono/mono/tree/0bcbe39b148bb498742fc68416f8293ccd350fb6/mcs/class/System.Windows.Forms/System.Windows.Forms

**Comment by thomaslevesque** [comment]
Indeed it does, but the .NET Core tooling doesn't know about it ;)

**Comment by TerribleDev** [comment]
Correct, My previous comment was to mention that other projects built with linux (like a xamarin app, which doesn't use netcore) could potentially have this issue. People have been using mono for a hell of a lot longer than dotnet core. in fact I [blogged about it in 2014](https://blog.terribledev.io/mono-not-just-a-sickness/). I would like to make sure we cover both platforms, not just net core. ;)

**Comment by subhendu-de** [comment]
Is there any support or workaround to circumvent the issue? Looks like StyleCop.MsBuild will not work with "dotnet build" command.
